src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -488,6 +488,9 @@ def load_command_table(self, _):
         g.custom_command('start', 'start_webapp')
         g.custom_command('restart', 'restart_webapp')
 
+    with self.command_group('logicapp deployment source') as g:
```

Comment:
in webapp & function app deployment source config-zip was used because it was supporting GitHub & other types. For logic app this is the only deployment method.. so this long command group makes no sense. call it az logicapp deploy - clarify in the help that this uses zip deployment & will perform remote build.